<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Lab1_3Class</class>
 <widget class="QMainWindow" name="Lab1_3Class">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>858</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Lab1_3</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupField">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>20</y>
      <width>221</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Создать поле</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_11">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>281</width>
       <height>71</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_11">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_23">
        <property name="text">
         <string>считать из файла</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="lineEdit_3">
        <property name="text">
         <string>output.txt</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="but_3">
        <property name="text">
         <string>создать</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupPres">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>290</y>
      <width>281</width>
      <height>281</height>
     </rect>
    </property>
    <property name="title">
     <string>Стенд</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_12">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>254</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_12">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item>
       <widget class="QLabel" name="label_24">
        <property name="text">
         <string>name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="comboPress_3"/>
      </item>
      <item>
       <widget class="QLineEdit" name="namePresLine_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_25">
        <property name="text">
         <string>radius</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="radiusPresLine_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_26">
        <property name="text">
         <string>coord</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <widget class="QLineEdit" name="coordPresX_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="coordPresY_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_17">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_18">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="but4_3">
          <property name="text">
           <string>отмена</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="but3_3">
          <property name="text">
           <string>добавить</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_18">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="but8_3">
        <property name="text">
         <string>удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="btnGroup">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>30</y>
      <width>271</width>
      <height>211</height>
     </rect>
    </property>
    <property name="title">
     <string>Меню</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_13">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>141</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_13">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_19">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="addFieldBut_3">
          <property name="text">
           <string>создать</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changeFieldBut_3">
          <property name="text">
           <string>изменить</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_19">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_27">
        <property name="text">
         <string>Машина</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_20">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="addCarBut_3">
          <property name="text">
           <string>добавить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changeCarBut_3">
          <property name="text">
           <string>изменить</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_20">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_28">
        <property name="text">
         <string>Стенд</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_21">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="addPresBut_3">
          <property name="text">
           <string>добавить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="changePressBut_3">
          <property name="text">
           <string>изменить</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_21">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="changeField">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>201</width>
      <height>271</height>
     </rect>
    </property>
    <property name="title">
     <string>Изменить поле</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_14">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>261</width>
       <height>157</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_14">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <item>
       <widget class="QPushButton" name="delAll_3">
        <property name="text">
         <string>удалить все</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delAllCars_3">
        <property name="text">
         <string>удалить все машины</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="delAllPres_3">
        <property name="text">
         <string>удалить все стенды</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_29">
        <property name="text">
         <string>печать в файл</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="strToFile_3"/>
      </item>
      <item>
       <widget class="QPushButton" name="printBtn_3">
        <property name="text">
         <string>печать</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupCar">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>90</y>
      <width>291</width>
      <height>341</height>
     </rect>
    </property>
    <property name="title">
     <string>Машина</string>
    </property>
    <widget class="QWidget" name="verticalLayoutWidget_15">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>271</width>
       <height>301</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_15">
      <property name="spacing">
       <number>6</number>
      </property>
      <property name="sizeConstraint">
       <enum>QLayout::SetMinAndMaxSize</enum>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QComboBox" name="comboBox_3"/>
      </item>
      <item>
       <widget class="QComboBox" name="comboBoxCar_3"/>
      </item>
      <item>
       <widget class="QLabel" name="label_30">
        <property name="text">
         <string>name</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="nameAddLine_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_31">
        <property name="text">
         <string>Coord</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_22">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <widget class="QLineEdit" name="coordXLine_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="coordYLine_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_22">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_32">
        <property name="text">
         <string>Gabarites</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_23">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <widget class="QLineEdit" name="gabXLine_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="gabYLine_3">
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_23">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="label_33">
        <property name="text">
         <string>Angle</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="angleLine_3">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_24">
        <property name="sizeConstraint">
         <enum>QLayout::SetMinAndMaxSize</enum>
        </property>
        <item>
         <widget class="QPushButton" name="but5_3">
          <property name="text">
           <string>отмена</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="but2_3">
          <property name="text">
           <string>добавить</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_24">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="but7_3">
        <property name="text">
         <string>удалить</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
   <widget class="QPushButton" name="reduceBut">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="lab1_3.qrc"/>
 </resources>
 <connections/>
</ui>
